const Error = require('./Error');

/**
 * @func Builder.Table
 * @contructor
 * @param {object} options
 * @throws {error}
 */
exports.Table = async (options={}) => {
  // await
};

/**
 * @func Builder.Global
 * @contructor
 * @param {object} options
 * @throws {error}
 */
exports.Global = async (options={}) => {
  // await
};

/**
 * @func Builder.Item
 * @contructor
 * @param {object} options
 * @throws {error}
 */
exports.Item = async (options={}) => {
  // await
};

/**
 * @func Builder.Query
 * @contructor
 * @param {object} options
 * @throws {error}
 */
exports.Query = async (options={}) => {
  // await
};

/**
 * @func Builder.Scan
 * @contructor
 * @param  {object} options
 * @throws {error}
 */
exports.Scan = async (options={}) => {
  // await
};

/**
 * @func Builder.Attribute
 * @contructor
 * @param  {object} options
 * @throws {error}
 */
exports.Attribute = async (options={}) => {
  // await
};

/**
 * @func Builder.Condition
 * @contructor
 * @param  {object} options
 * @throws {error}
 */
exports.Condition = async (options={}) => {
  // await
};

/**
 * @func Builder.Tag
 * @contructor
 * @param  {object} options
 * @throws {error}
 */
exports.Tag = async (options={}) => {
  // await
};
